src/util/report-migration.xml

```diff
@@ -43,6 +43,36 @@
           </ul>
         </details></li>
       </t>
+      <t t-elif="category == 'Export Templates'">
```

Comment:
With your example. Weren't those hypothetical export routines broken anyway? (we didn't update them correctly). If yes, then my point stands: we aren't doing anything new, it may be more confusing to start reporting back this. In any case, I'd expect people relying in export templates to test them during the upgrade testing phase.

Anyway, for an extra opinion let's ask @KangOl :)

Reply:
> Weren't those hypothetical export routines broken anyway?
Yes. In the cases I've seen firsthand, the implementation before this PR just leaves the broken references for removed fields, leading to a UI traceback when loading the template for export.

Reply:
I've pushed suggested changes with all reports stuff removed.